include/admin/images.inc.php

```diff
@@ -20,13 +20,6 @@
 if (!is_array($serendipity['POST']) && $serendipity['GET']['adminAction'] == 'multicheck') {
     unset($serendipity['GET']['adminAction']);
 }
-// Listens on toggle_dir STRICT to list items per directory, or include all sub directory items
```

Comment:
This is still evaluated in the media_pane.tpl file?!

Reply:
I wasn't sure about this! It was my understanding that the `toggle_dir` set here is not evaluated at all, it does nothing. I might have missed where it's applied?

Reply:
Right, I missed it. Will be part of the new backend pr.